packages/amplify-cli/src/extensions/amplify-helpers/yaml-cfn.ts

```diff
@@ -0,0 +1,59 @@
+import * as yaml from 'yaml';
```

Comment:
Why do we need this file? Can this be re-used from somewhere?

Reply:
Will replace this with yaml library call.